File: 프로그래머스/0/120834. 외계행성의 나이/외계행성의 나이.java
class Solution {
    public String solution(int age) {
        StringBuffer answer = new StringBuffer();
        String tmp = String.valueOf(age);
        for (int i = 0; i < tmp.length(); i++) {
            char c = (char) ((int) tmp.charAt(i) + 49);
            answer.append(c);
        }
        return answer.toString();
    }
}

File: 프로그래머스/0/120834. 외계행성의 나이/외계행성의 나이.swift
import Foundation

func solution(_ age:Int) -> String {
    var answer = ""
    let alienAge = [
            "0": "a",
            "1": "b",
            "2": "c",
            "3": "d",
            "4": "e",
            "5": "f",
            "6": "g",
            "7": "h",
            "8": "i",
            "9": "j"
        ]
        
    for i in String(age) {
        answer += alienAge[String(i)]!
    }
    
    return answer
}